### PR TITLE
chore: bump lerna version

### PR DESCRIPTION
**Pull request by lime-green**

### CODE REVIEW

File: src/command-helpers.js [hunk 1]
```diff
@@ -53,7 +53,8 @@ async function lernaPublish({ registryUrl = null } = {}) {
     const command = [
         lernaExecutable,
         'publish',
-        '--skip-git',
+        '--no-git-tag-version',
```
Comment (noahnu) [inline src/command-helpers.js]: how does this affect fe-libraries?

Comment (lime-green) [reply src/command-helpers.js]: It shouldn’t, they’ll be on an old release?

**Comment by noahnu** [comment]
@lime-green Is there semantic versioning + semantic commits on this repo?

**Comment by msrose** [comment]
@all-contributors please add @lime-green for code and infra

**Pull request merged**